azure-pipelines.yml

```diff
@@ -7,37 +7,20 @@ trigger:
 - master
```

Comment:
This file can actually be removed as we GH actions should be enough.

Reply:
done!